### ISSUE TITLE
clickListener for lyrics

### ISSUE THREAD

**Issue by SlowBubble** [paulrosen/abcjs#523]
I am trying out the awesome abcjs editor at https://www.abcjs.net/abcjs-editor.html.
It seems the clickListener does not listen to clicks on the lyrics. Is that expected?

**Comment by SlowBubble** [comment]
This may be related: the click event does not distinguish between the click on the chord symbol and the click on the note associated with the chord.

**Comment by paulrosen** [comment]
That's true - I'm planning on breaking the different parts of the note apart for more fine-grained targeting. That will be soon, but I don't have a date.

You can set it up to click on lots of different things with the `selectTypes` parameter. By default it only selects notes, rests, and bar lines.

Here is more information: https://paulrosen.github.io/abcjs/visual/dragging.html#specifying-what-is-selectable

While looking this over, I realized that lyrics are not included as a selectable item. That is a bug and I'll have that fixed in the next release.

**Comment by SlowBubble** [comment]
Good to know, thanks!
